pkg/cmd/time-entry/delete/delete.go

```diff
@@ -40,9 +40,17 @@ func NewCmdDelete(f cmdutil.Factory) *cobra.Command {
 			$ %[1]s current
 			# no output
 
+			# deleting the last time entry
+			$ %[1]s last
+			# no output
+
 			# deleting multiple time entries
 			$ %[1]s 62b5b51085815e619d7ae18d 62b5d55185815e619d7af928
 			# no output
+
+			# deleting last two entries
+			$ %[1]s last last
```

Comment:
i think this is a little bit dangerous

Reply:
i understand it works... but i wouldn't encourage it

Reply:
Got it. Removed that part.